go.mod

```diff
@@ -149,3 +149,5 @@ require (
 )
 
 replace sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.13.1-0.20230315234915-a26de2d610c3
+
+replace github.com/vishvananda/netns => github.com/alban/netns v0.0.5-0.20230524185006-155d84c555d6
```

Comment:
What about moving that fork into this organization?

Reply:
Ok!